apps/docs/content/docs/editor.mdx

```diff
@@ -350,6 +350,109 @@ The camera may be in two states, `idle` or `moving`.
 
 You can get the current camera state with [Editor#getCameraState](?).
 
+# Bindings
```

Comment:
Would this section be better positioned in the Shapes article?

Reply:
I think it could even stand as an article of its own, but instructions from @steveruizok were to just put everything in the editor article